src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1410,6 +1410,8 @@ CHIP_ERROR CASESession::SendSigma3a()
             SuccessOrExit(err = helper->ScheduleWork());
             mSendSigma3Helper = helper;
             mExchangeCtxt->WillSendMessage();
+            mExchangeCtxt->SetShouldPeerBeActive(
+                true); // When we send sigma3, peer has processed sigma 1 and sent sigma2 which means it is active
```

Comment:
"it is active" --> "it is likely active"

Reply:
done